src/main/java/frc/robot/subsystems/superstructure/arm/Arm.java

```diff
@@ -132,19 +131,15 @@ public void periodic() {
     }
 
     private boolean atPositionSetpoint() {
-        return setpoint.atSetpoint(inputs.leftPivotPositionRots, inputs.leftPivotVelocityRotsPerSec)
```

Comment:
This should probably still use both positions, or perhaps at least the left position as it will be the one that is fused (see comment about that later).

Reply:
It does. See my comment further down - getting the position from the cancoder is JUST the cancoder position, it is **not** fused.